ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java

```diff
@@ -589,6 +589,8 @@ private void handleJsonRPCRequest(final RoutingContext routingContext) {
               token,
               user -> handleJsonBatchRequest(routingContext, array, user));
         }
+      } catch (final NullPointerException ex) {
+        handleJsonRpcError(routingContext, null, JsonRpcError.PARSE_ERROR);
```

Comment:
Is there a more appropriate error we can give that's related to incorrect request header?

Reply:
it's not necessarily the request header though